.appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: "{build}"
+
+os: Visual Studio 2017
+
+platform:
+  - x64
+
+build:
+  parallel: true  # enable MSBuild parallel builds
+  verbosity: minimal
+
+install:
+  - if not exist llvm.tar.xz appveyor DownloadFile "https://ziglang.org/deps/llvm+clang-7.0.0-win64-msvc-release.tar.xz" -FileName llvm.tar.xz
```

Comment:
Question: is this the best you can find? Will it download the archive again and again?